include/IDriverFence.h

```diff
@@ -133,7 +133,7 @@ class GPUEventWrapper : public core::Uncopyable
             return *this;
         }
 
-        template<class Clock=std::chrono::steady_clock, class Duration=typename Clock::duration>
+        template<class Clock=std::chrono::high_resolution_clock, class Duration=typename Clock::duration>

```

Comment:
sure this isnt a problem? canging the type?

Reply:
steady_clock is best, its the only one that has guarantees that time wont go backwards (due to clock change)